Convert opengear to YAML-based discovery, add some new sensors, add test data

martijn-schmidt opened the pull request:
I've converted the PHP-based discovery for the opengear signal sensor to YAML-based, added some new sensors, and added test data. And it all worked without having to change PHP code for the YAML discovery functions - sometimes life is simple! :)

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

laf commented on includes/definitions/discovery/opengear.yaml:
```diff
@@ -0,0 +1,65 @@
+                    value: ogCellModem3gRssi
+                    num_oid: '.1.3.6.1.4.1.25049.17.17.1.11.{{ $index }}'
+                    descr: 'Cellular 3G Modem RSSI'
+                    index: 'ogCellModem3gRssi.{{ $index }}'
```
To avoid people losing data, `index` should be `11.{{ $index }}`

martijn-schmidt replied:
Done

laf commented on includes/definitions/discovery/opengear.yaml:
```diff
@@ -0,0 +1,65 @@
+                    value: ogCellModem4gRssi
+                    num_oid: '.1.3.6.1.4.1.25049.17.17.1.12.{{ $index }}'
+                    descr: 'Cellular 4G Modem RSSI'
+                    index: 'ogCellModem4gRssi.{{ $index }}'
```
To avoid people losing data, `index` should be `12.{{ $index }}`

martijn-schmidt replied:
Done

martijn-schmidt commented:
@laf thanks for the feedback, I've modified the YAML for those two sensors to use numbered rather than named indexes and also updated the test data to match this change.

murrant commented:
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-55-release-changelog-august-2019/9428/1
